server/game/cards/01_SOR/units/WedgeAntillesStarOfTheRebellion.ts

```diff
@@ -0,0 +1,26 @@
+import AbilityHelper from '../../../AbilityHelper';
+import { NonLeaderUnitCard } from '../../../core/card/NonLeaderUnitCard';
+import { KeywordName, RelativePlayer, Trait } from '../../../core/Constants';
+
+export default class WedgeAntillesStarOfTheRebellion extends NonLeaderUnitCard {
+    protected override getImplementationId() {
+        return {
+            id: '4339330745',
+            internalName: 'wedge-antilles#star-of-the-rebellion',
+        };
+    }
+
+    public override setupCardAbilities() {
+        this.addConstantAbility({
+            title: 'Each friendly VEHICLE unit gets +1/+1 and gains Ambush.',
+            targetController: RelativePlayer.Self,
+            matchTarget: (card) => card.hasSomeTrait(Trait.Vehicle),
```

Comment:
It might be worth adding `cardTypeFilter: WildcardCardType.Unit` just to be safe. Seems pretty unlikely that an upgrade with the Vehicle trait would be printed but you never know

Reply:
Makes sense. Adding.